resolwe/flow/management/commands/clean_test_dir.py

```diff
@@ -0,0 +1,53 @@
+""".. Ignore pydocstyle D400.
```

Comment:
To follow Django's naming convention, this file should be named `cleantestdir.py`.